upgrade.json

```diff
@@ -26,7 +26,8 @@
   "23": {
     "slave": ["20160516-mbs-8838.sql",
               "20160516-mbs-8720.sql",
-              "20141226-geodesic-index.sql"],
-    "standalone": []
+              "20141226-geodesic-index.sql",
+              "20160310-mbs-4501-alternative-tracklists.sql"],
+    "standalone": ["20160310-mbs-4501-alternative-tracklists-fks.sql"]
```

Comment:
How do these changes with `slave`, `slave_only`, and `standalone` work? Don't think I asked about that before.

Also why are foreign keys created only in `standalone`?